Basic-Scripts/balanced_paranthesis.py

```diff
@@ -0,0 +1,26 @@
+open_bracket_list = ["[", "{", "("]
+close_bracket_list = ["]", "}", ")"]
+
+
+def checkParanthsis(data):

```

Comment:
Include docstring documentation for the functions.

Reply:
Done